wallet/db.c

```diff
@@ -215,7 +214,7 @@ static struct migration dbmigrations[] = {
      * invoices to current time. */
     {SQL("ALTER TABLE invoices ADD paid_timestamp INTEGER;"), NULL},
     {SQL("UPDATE invoices"
-	 "   SET paid_timestamp = strftime('%s', 'now')"
+	 "   SET paid_timestamp = 1"
```

Comment:
Fair enough, though we could argue this belongs in the postgres translation, this is good enough since it will Never Happen.

Reply:
Yep, that was my reason for just setting a dummy value, nobody should have a DB this old, and nothing bad happens if it goes wrong.

Reply:
Just to make sure I went and looked up where the migration was introduced: https://github.com/ElementsProject/lightning/commit/a88c73a41b9b4f5c43b8348e5247bd22578d0042

This predates v0.6 as well.